.circleci/config.yml

```diff
@@ -110,12 +123,15 @@ jobs:
             sudo docker-php-ext-enable imagick
             sudo docker-php-ext-install mysqli
             sudo apt-get install mariadb-client
+            curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
+            chmod +x wp-cli.phar
+            sudo mv wp-cli.phar /usr/local/bin/wp
       - run:
           name: "Run Tests"
           command: |
-            bash bin/install-wp-tests.sh wordpress_test root '' 127.0.0.1 latest
-            composer phpunit
-            WP_MULTISITE=1 composer phpunit
-            rm -rf $WP_TESTS_DIR $WP_CORE_DIR
-            bash bin/install-wp-tests.sh wordpress_test root '' 127.0.0.1 nightly true
+            composer test:install:withdb
             composer phpunit
+      - run:
+          name: "Run SimpleSAMLphp tests"
+          command: |
+            IS_CI="true" bash bin/test-all-versions.sh
```

Comment:
Are these scripts actually making an assertions?

Reply:
yes, and other stuff. 
linsey wrote a script that runs a series of setup/teardowns for different simplesamlphp versions and phpunit tests.